index.md

```diff
@@ -67,6 +67,41 @@ function callback(data) {
 
     L.control.scale({maxWidth: 300}).addTo(map);
 
+    new L.Control.BootstrapDropdowns({
+        position: "topright",
+        className: "menu",
+        autoClose: "outside",
+        menuItems: [
+            {
+                html: '<i class="fa-solid fa-map-marked-alt"></i> Run the GitHub actions below',
+                title: "Open Street Map",
+                current: true,
+                disabled: true,
+            },
+            {
+                separator: true,
+            },
+            {
+                html: '<i class="fa-solid fa-user-plus"></i> Add yourself to the map',
```

Comment:
These icons are cool. Fontawesome is also the size of leaflet :disappointed:.  If I had to choose one, I would rather choose Fontawesome rather than bootstrap as it is (a bit) smaller :shrug: